metricbeat/module/kibana/status/_meta/fields.yml

```diff
@@ -23,7 +23,31 @@
     - name: status.overall.state
       type: keyword
       description: >
-        Kibana overall state.
+        Kibana overall state (v7 format).
```

Comment:
The fields are not updated in the docs: [fields.asciidoc
](https://github.com/elastic/beats/blob/main/metricbeat/docs/fields.asciidoc#status-1) - I tried running `make update` and it adds them there.